html/index.html

```diff
@@ -3,9 +3,213 @@
+        <img src='#' alt='#' />
+        <p>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nam perferendis fugit doloribus qui magni ratione
+          nisi assumenda officiis explicabo asperiores dolorem ipsam voluptatibus libero harum consectetur et, inventore
+          commodi cum.
+        </p>
+        <ol>
+          <li>Directing</li>
+          <li>Editing</li>
+          <li>Lighting</li>
+          <li>Production Assistant</li>
+        </ol>
+        <button>Contact Me</button>
+      </div>
+      <div>
+        <h3>Acting</h3>
+        <img src='#' alt='#' />
+        <p>
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Enim itaque adipisci alias sed atque quas dolore,
+          neque quasi nulla eos quaerat quae tempore amet laboriosam delectus aspernatur voluptatibus. Suscipit,
+          placeat!
+        </p>
+        <button>Contact Me</button>
+      </div>
+    </div>
+    <div>
+      <h2>Gallery</h2>
+      <div>
+        <h3>Videos</h3>
+        <img src='#' alt='#' />
+        <img src='#' alt='#' />
+        <img src='#' alt='#' />
+        <img src='#' alt='#' />
+        <p>See More Videos</p>
+      </div>
+      <div>
+        <h3>Videos</h3>
+        <img src='#' alt='#' />
+        <img src='#' alt='#' />
+        <img src='#' alt='#' />
+        <img src='#' alt='#' />
+        <p>See More Photos</p>
+      </div>
+    </div>
+    <div>
+      <h2>About</h2>
+      <div>
+        <h3>Entrepreneur</h3>
+        <img src='#' alt='#' />
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate, possimus dolores culpa repellendus
+          perferendis, porro iste corporis quam aliquam, aliquid beatae assumenda mollitia! Possimus, porro dolore
+          cupiditate ipsam quos explicabo!
+        </p>
+      </div>
+      <div>
+        <h3>Producer</h3>
+        <img src='#' alt='#' />
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate, possimus dolores culpa repellendus
+          perferendis,
+          porro iste corporis quam aliquam, aliquid beatae assumenda mollitia! Possimus, porro dolore cupiditate ipsam
+          quos
+          explicabo!
+        </p>
+      </div>
+      <div>
+        <h3>Actor</h3>
+        <img src='#' alt='#' />
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate, possimus dolores culpa repellendus
+          perferendis,
+          porro iste corporis quam aliquam, aliquid beatae assumenda mollitia! Possimus, porro dolore cupiditate ipsam
+          quos
+          explicabo!
+        </p>
+      </div>
+    </div>
+    <div>
+      <h2>Contact</h2>
+      <div>
+        <p>email</p>
+        <p>phone</p>
+        <p>phone</p>
+      </div>
+      <form>
+        <label>Name</label>
+        <input />
+        <label>Email</label>
+        <input />
+        <label>Services</label>
+        <input />
```

Comment:
this should be a `<select>` because it will be a drop down of all of Adam's services.

Services:
- corporate workshops team building
- improv classes
- tv/film production
- acting